src/global_config.cc

```diff
@@ -0,0 +1,37 @@
+/*!
+ * Copyright 2020 by Contributors
+ * \file global_config.cc
+ * \brief Global configuration for XGBoost
+ * \author Hyunsu Cho
+ */
+
+#include <dmlc/thread_local.h>
+#include "xgboost/global_config.h"
+
+namespace xgboost {
+
+void GlobalConfiguration::SetArgs(const Args& args) {
+  for (const auto& e : args) {
+    if (e.first == "verbosity") {
```

Comment:
I think we don't need to check for str equality.  The `XGBoostParameter` struct can return all unknown parameters and consume all known parameters automatically.

Reply:
Do you want me to create a new `XGBoostParameter` struct? I didn't do that because I thought you didn't like duplicating parameters. If I create `GlobalConfigurationParams` struct, it will duplicate `verbosity` parameter.

Reply:
Actually duplicating parameters may be unavoidable, since we are already doing that in the Python layer. Let me go ahead and create `GlobalConfigurationParams`.

Reply:
Please don't duplicate parameters.  We can have a deeper look into it later.

Reply:
Got it. I won't create `GlobalConfigurationParams`